openadapt/config.py

```diff
@@ -99,6 +99,7 @@ def getenv_fallback(var_name):
     locals()[key] = val
 
 ROOT_DIRPATH = pathlib.Path(__file__).parent.parent.resolve()
+DB_FNAME = getenv_fallback("DB_FNAME")
```

Comment:
Shouldn't this be defined already above? Can you please clarify why we are we explicitly redefining it here?

Reply:
It's defined, but every linter keeps telling me it's undefined so it's mainly there just to silence the warnings:
![image](https://github.com/MLDSAI/OpenAdapt/assets/57018940/f3a1c35c-30fb-4b01-b739-15b83680b39f)

Reply:
What do you think about just adding a `#noqa` directive (or similar) to suppress the warning?

Reply:
Even I got this lint warning for DB_FNAME, but I always ignore it

Reply:
Yes, that worked.